### PR TITLE
brings in FV3 "the dycore" updated to be consistent with the latest public release 

### LINKED ISSUE #375: update FV3 "the dycore" to fully support advanced nesting in regional domains
## Description
need to update FV3 "the dycore" to the latest version available via NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/128

## Solution
merge in PR when available

## Alternatives
N/A

## Testing:

1. Have you tested the code changes?  yes on gaea
2. Have you run regression test in ufs-weather-model or ufs-s2s-model with code changes?  yes
- Will the baseline results change?  yes
- If yes, please give brief explanation of your code updates on the regression test.
The change to how the VMR and MMR tracers are handled for ICs created by chgres from different sources
Bug fixes to regional boundary condition handling (surface pressure and wrong array extents for a staggered grid variable)

## Dependent PRs:

Directly reference any issues or PRs in this or other repositories that this is related to, and describe how they are related. Example:
- required to support noaa-emc/GFDL_atmos_cubed_sphere/pull/128
- associated ufs-weather-model/issue/760

### LINKED ISSUE #760: advanced nesting support for regional domains from FV3 "the dycore"
## Description
Need to bring in the latest public release of FV3 "the dycore" that is awaiting merge into dev/emc branch in NOAA-GFDL/GFDL_atmos_cubed_sphere (PR #128)

## Solution
Complete review of NOAA-GFDL/GFDL_atmos_cubed_sphere PR #128

## Alternatives
N/A

## Related to
Directly reference any issues or PRs in this or other repositories that this is related to, and describe how they are related. 
 - all PRs needed for this issue are yet to be created

### PR DESCRIPTION
**Pull request by bensonr**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [x] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

This PR brings in an update for fv3atm that contains updates to the latest public release of the FV3 dycore encapsulated in PR [#128](https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/128).

This PR also disables the control_csawmg test for gaea.intel due to a numerical instability brought about by the updates to the dycore.  The same test runs to completion on hera.intel.

This PR requires a new datestamp for the RTS ICs.  The particular updates are to the gdas tests which pass regression test for this update.

This PR changes answers for all regional tests and any test with ICs created by chgres from sources that sets the data_source_fv3gfs=.true. option in FV3/atmos_cubed_sphere/tools/external_ic.F90::get_nggps_ic routine.  Therefore new baselines will need to be created.

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #760 
- fixes noaa-emc/fv3atm/issues/375

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [ ] jet.intel (not accessible)
- [x] wcoss_cray
- [x] wcoss_dell_p3
- [x] CI: https://github.com/ufs-community/ufs-weather-model/actions/runs/1171261327

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

bensonr/GFDL_atmos_cubed_sphere/tree/master2emc
bensonr/fv3atm/tree/dycore

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/128
- waiting on noaa-emc/fv3atm/pull/377

### CODE REVIEW

**Comment by junwang-noaa** [comment]
Rusty identified following dycore code updates will cause baselines to change:
```
diff --git a/model/fv_regional_bc.F90 b/model/fv_regional_bc.F90
index 08f7c61..0ed9845 100644
--- a/model/fv_regional_bc.F90
+++ b/model/fv_regional_bc.F90
@@ -1271,7 +1270,0 @@ contains
-
-      do j=jsd,jed
-      do i=isd,ied
-        ps0(i,j) = Atm%ps(i,j)
-      enddo
-      enddo
-
@@ -1299,6 +1291,0 @@ contains
-
-      do j=jsd,jed
-      do i=isd,ied
-        Atm%ps(i,j) = ps0(i,j)
-      enddo
-      enddo
@@ -1417 +1404 @@ contains
-      ak_in(1) = max(1.e-9, ak_in(1))
+      ak_in(1) = 1.e-9
@@ -3401 +3388 @@ contains
-      allocate(BC_side%divgd_BC(is_we:ie_we,js_sn:je_sn,klev)) ; BC_side%divgd_BC=real_snan
+      allocate(BC_side%divgd_BC(is_0:ie_0,js_0:je_0,klev)) ; BC_side%divgd_BC=real_snan
@@ -4403 +4390 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v')then
@@ -4412 +4399 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v')then
@@ -4435 +4422 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v')then
@@ -4441 +4428 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc')then
@@ -4448 +4435 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v')then
@@ -4452 +4439 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc')then
@@ -4476 +4463 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='u')then
@@ -4489 +4476 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-              if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='divgd')then
+              if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc')then
@@ -4504 +4491 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc')then
@@ -4527 +4514 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='u')then
@@ -4534 +4521 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='uc'.or.trim(bc_vbl_name)=='v')then
@@ -4544 +4531 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-              if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='divgd')then
+              if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc')then
@@ -4559 +4546 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc'.or.trim(bc_vbl_name)=='divgd')then
+            if(trim(bc_vbl_name)=='u'.or.trim(bc_vbl_name)=='vc')then
@@ -5040,0 +5028 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
+              bc_side_t0%divgd_BC(i,j,k)=bc_side_t1%divgd_BC(i,j,k)
@@ -5080,6 +5068 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
-
-            do j=js_s,je_s
-            do i=is_w,ie_w
-              bc_side_t0%divgd_BC(i,j,k)=bc_side_t1%divgd_BC(i,j,k)
-            enddo
-            enddo
+          enddo
@@ -5086,0 +5070 @@ subroutine remap_scalar_nggps_regional_bc(Atm                         &
+          do k=1,nlev
diff --git a/tools/external_ic.F90 b/tools/external_ic.F90
index 443037c..a28e97e 100644
--- a/tools/external_ic.F90
+++ b/tools/external_ic.F90
@@ -798,5 +798,5 @@ contains
-            qt = wt/(1. - (Atm%q(i,j,k,liq_wat) + &
-                           Atm%q(i,j,k,ice_wat) + &
-                           Atm%q(i,j,k,rainwat) + &
-                           Atm%q(i,j,k,snowwat) + &
-                           Atm%q(i,j,k,graupel)))
+            qt = wt*(1. + Atm%q(i,j,k,liq_wat) + &
+                          Atm%q(i,j,k,ice_wat) + &
+                          Atm%q(i,j,k,rainwat) + &
+                          Atm%q(i,j,k,snowwat) + &
+                          Atm%q(i,j,k,graupel))
@@ -804 +804 @@ contains
-            qt = wt/(1. - sum(Atm%q(i,j,k,2:Atm%flagstruct%nwat)))
+            qt = wt*(1. + sum(Atm%q(i,j,k,2:Atm%flagstruct%nwat)))
```

**Comment by bensonr** [comment]
@junwang-noaa - thanks for including the diff stats.  Please be aware the new/modified code is denoted by a '-' and the original code is the '+' in the diff listed above.

**Comment by BrianCurtis-NOAA** [comment]
Machine: cheyenne
Compiler: gnu
Job: BL
Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/720092158/20210826073009/ufs-weather-model
Please manually delete: /glade/scratch/dtcufsrt/FV3_RT/rt_10056
Baseline creation and move successful
Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/720092158/20210826075727/ufs-weather-model
Please manually delete: /glade/scratch/dtcufsrt/FV3_RT/rt_45380
Test fv3_esg_HAFS_v0_hwrf_thompson 015 failed failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 015 failed in run_test failed 
Test control_ugwpv1 009 failed failed 
Test control_ugwpv1 009 failed in run_test failed 
Test control_thompson 007 failed failed 
Test control_thompson 007 failed in run_test failed 
Test control 001 failed failed 
Test control 001 failed in run_test failed 
Test control_thompson_no_aero 008 failed failed 
Test control_thompson_no_aero 008 failed in run_test failed 
Test fv3_gsd 011 failed failed 
Test fv3_gsd 011 failed in run_test failed 
Test control_stochy 004 failed failed 
Test control_rrtmgp 006 failed failed 
Test control_stochy 004 failed in run_test failed 
Test control_rrtmgp 006 failed in run_test failed 
Test control_flake 005 failed failed 
Test control_flake 005 failed in run_test failed 
Test control_ras 010 failed failed 
Test control_ras 010 failed in run_test failed 
Test control_ras_debug 028 failed failed 
Test control_ras_debug 028 failed in run_test failed 
Test control_noahmp_debug 029 failed failed 
Test control_noahmp_debug 029 failed in run_test failed 
Test control_debug 016 failed failed 
Test control_debug 016 failed in run_test failed 
Test control_diag_debug 017 failed failed 
Test control_diag_debug 017 failed in run_test failed 
Test fv3_gsd_debug 021 failed failed 
Test fv3_gsd_debug 021 failed in run_test failed 
Test control_c48 003 failed failed 
Test control_c48 003 failed in run_test failed 
Test fv3_rrfs_v1beta_debug 020 failed failed 
Test fv3_rrfs_v1beta_debug 020 failed in run_test failed 
Test control_ugwpv1_debug 027 failed failed 
Test control_ugwpv1_debug 027 failed in run_test failed 
Test control_thompson_extdiag_debug 025 failed failed 
Test control_thompson_extdiag_debug 025 failed in run_test failed 
Test control_thompson_debug 023 failed failed 
Test control_thompson_debug 023 failed in run_test failed 
Test control_rrtmgp_debug 026 failed failed 
Test control_rrtmgp_debug 026 failed in run_test failed 
Test fv3_gsd_diag_debug 022 failed failed 
Test fv3_gsd_diag_debug 022 failed in run_test failed 
Test fv3_rrfs_v1alpha_debug 019 failed failed 
Test control_thompson_no_aero_debug 024 failed failed 
Test fv3_rrfs_v1alpha_debug 019 failed in run_test failed 
Test control_thompson_no_aero_debug 024 failed in run_test failed 
Please make changes and add the following label back:
cheyenne-gnu-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: gaea
Compiler: intel
Job: BL
Repo location: /lustre/f2/pdata/ncep/emc.nemspara/autort/pr/720092158/20210826133005/ufs-weather-model
Please manually delete: /lustre/f2/scratch/emc.nemspara/FV3_RT/rt_26714
Test control_thompson_no_aero 030 failed failed 
Test control_thompson_no_aero 030 failed in run_test failed 
Test control_csawmgt 025 failed failed 
Test control_csawmgt 025 failed in run_test failed 
Test regional_quilt_hafs 035 failed failed 
Test regional_quilt 034 failed failed 
Test regional_quilt 034 failed in run_test failed 
Test regional_quilt_hafs 035 failed in run_test failed 
Test control_c384 016 failed failed 
Test control_c384 016 failed in run_test failed 
Please make changes and add the following label back:
gaea-intel-BL

**Comment by climbfuji** [comment]
> Machine: cheyenne
> Compiler: gnu
> Job: BL
> Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/720092158/20210826073009/ufs-weather-model
> Please manually delete: /glade/scratch/dtcufsrt/FV3_RT/rt_10056
> Baseline creation and move successful
> Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/720092158/20210826075727/ufs-weather-model
> Please manually delete: /glade/scratch/dtcufsrt/FV3_RT/rt_45380
> Test fv3_esg_HAFS_v0_hwrf_thompson 015 failed failed
> Test fv3_esg_HAFS_v0_hwrf_thompson 015 failed in run_test failed
> Test control_ugwpv1 009 failed failed
> Test control_ugwpv1 009 failed in run_test failed
> Test control_thompson 007 failed failed
> Test control_thompson 007 failed in run_test failed
> Test control 001 failed failed
> Test control 001 failed in run_test failed
> Test control_thompson_no_aero 008 failed failed
> Test control_thompson_no_aero 008 failed in run_test failed
> Test fv3_gsd 011 failed failed
> Test fv3_gsd 011 failed in run_test failed
> Test control_stochy 004 failed failed
> Test control_rrtmgp 006 failed failed
> Test control_stochy 004 failed in run_test failed
> Test control_rrtmgp 006 failed in run_test failed
> Test control_flake 005 failed failed
> Test control_flake 005 failed in run_test failed
> Test control_ras 010 failed failed
> Test control_ras 010 failed in run_test failed
> Test control_ras_debug 028 failed failed
> Test control_ras_debug 028 failed in run_test failed
> Test control_noahmp_debug 029 failed failed
> Test control_noahmp_debug 029 failed in run_test failed
> Test control_debug 016 failed failed
> Test control_debug 016 failed in run_test failed
> Test control_diag_debug 017 failed failed
> Test control_diag_debug 017 failed in run_test failed
> Test fv3_gsd_debug 021 failed failed
> Test fv3_gsd_debug 021 failed in run_test failed
> Test control_c48 003 failed failed
> Test control_c48 003 failed in run_test failed
> Test fv3_rrfs_v1beta_debug 020 failed failed
> Test fv3_rrfs_v1beta_debug 020 failed in run_test failed
> Test control_ugwpv1_debug 027 failed failed
> Test control_ugwpv1_debug 027 failed in run_test failed
> Test control_thompson_extdiag_debug 025 failed failed
> Test control_thompson_extdiag_debug 025 failed in run_test failed
> Test control_thompson_debug 023 failed failed
> Test control_thompson_debug 023 failed in run_test failed
> Test control_rrtmgp_debug 026 failed failed
> Test control_rrtmgp_debug 026 failed in run_test failed
> Test fv3_gsd_diag_debug 022 failed failed
> Test fv3_gsd_diag_debug 022 failed in run_test failed
> Test fv3_rrfs_v1alpha_debug 019 failed failed
> Test control_thompson_no_aero_debug 024 failed failed
> Test fv3_rrfs_v1alpha_debug 019 failed in run_test failed
> Test control_thompson_no_aero_debug 024 failed in run_test failed
> Please make changes and add the following label back:
> cheyenne-gnu-BL

Weird. It says baseline creation was successful, and it looks like it was. For the verification step, I see the following error for the failing tests:
```
0: in fcst run phase 2, na=           0
144: line          570 NetCDF: HDF error
144:MPT ERROR: Rank 144(g:144) is aborting with error code 1.
144:    Process ID: 26133, Host: r10i6n26, Program: /glade/scratch/dtcufsrt/FV3_RT/rt_45380/control_noahmp_debug/fv3.exe
144:    MPT Version: HPE MPT 2.22  03/31/20 15:59:10
144:
144:MPT: --------stack traceback-------
```

Not sure if this was a temporary issue, a disk space problem or what. Will try to run the verification step manually.

**Comment by junwang-noaa** [comment]
On gaea, all the 4 failed tests were timed out. Will manually resubmit the
4 tests for baseline creation.

**Comment by climbfuji** [comment]
> On gaea, all the 4 failed tests were timed out. Will manually resubmit the 4 tests for baseline creation.
> […](#)

@bensonr Do we expect an increase in runtime from the dycore update?

**Comment by bensonr** [comment]
>
> On gaea, all the 4 failed tests were timed out. Will manually resubmit the
> 4 tests for baseline creation.
> … <#m_8415720038615092642_>
>
> @bensonr <https://github.com/bensonr> Do we expect an increase in runtime
> from the dycore update?
>

No I don't.  I was able to run a full set of tests without fail on gaea and
both intel and gnu on hera last night.

**Comment by BrianCurtis-NOAA** [comment]
Machine: cheyenne
Compiler: intel
Job: BL
Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/720092158/20210826074509/ufs-weather-model
Please manually delete: /glade/scratch/dtcufsrt/FV3_RT/rt_67736
Test fv3_gsd_diag_debug 065 failed in run_test failed 
Test datm_control_cfsr 075 failed in run_test failed 
Test compile 013 failed failed 
Test datm_control_gefs 076 failed in run_test failed 
Test datm_control_iau_gefs 077 failed in run_test failed 
Test datm_bulk_cfsr 078 failed in run_test failed 
Test datm_bulk_gefs 079 failed in run_test failed 
Test datm_mx025_cfsr 080 failed in run_test failed 
Test datm_mx025_gefs 081 failed in run_test failed 
Test cpld_control_c192 003 failed failed 
Test cpld_control_c192 003 failed in run_test failed 
Test regional_quilt 034 failed failed 
Test regional_quilt 034 failed in run_test failed 
Test cpld_bmark_v16 005 failed failed 
Test cpld_bmark_v16 005 failed in run_test failed 
Test datm_cdeps_bulk_cfsr 085 failed in run_test failed 
Test datm_cdeps_bulk_gefs 086 failed in run_test failed 
Test datm_cdeps_mx025_cfsr 087 failed in run_test failed 
Test datm_cdeps_mx025_gefs 088 failed in run_test failed 
Test datm_cdeps_debug_cfsr 089 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 044 failed failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 044 failed in run_test failed 
Test cpld_control_wave 008 failed failed 
Test cpld_control_wave 008 failed in run_test failed 
Test cpld_bmark_wave_v16 007 failed failed 
Test cpld_bmark_wave_v16 007 failed in run_test failed 
Test cpld_bmark_v16_nsst 006 failed failed 
Test cpld_bmark_v16_nsst 006 failed in run_test failed 
Test compile 019 failed failed 
Test datm_debug_cfsr 082 failed failed 
Test datm_debug_cfsr 082 failed in run_test failed 
Test datm_cdeps_control_cfsr 083 failed failed 
Test datm_cdeps_control_cfsr 083 failed in run_test failed 
Test control_rrtmgp 023 failed failed 
Test control_rrtmgp 023 failed in run_test failed 
Test regional_quilt_hafs 035 failed failed 
Test regional_quilt_hafs 035 failed in run_test failed 
Test control_csawmg 024 failed failed 
Test control_csawmg 024 failed in run_test failed 
Test compile 014 failed failed 
Test datm_cdeps_control_gefs 084 failed failed 
Test datm_cdeps_control_gefs 084 failed in run_test failed 
Test regional_quilt_RRTMGP 037 failed failed 
Test regional_quilt_RRTMGP 037 failed in run_test failed 
Test regional_quilt_netcdf_parallel 036 failed failed 
Test regional_quilt_netcdf_parallel 036 failed in run_test failed 
Test control_csawmgt 025 failed failed 
Test control_csawmgt 025 failed in run_test failed 
Test control_flake 026 failed failed 
Test control_ugwpv1 027 failed failed 
Test control_flake 026 failed in run_test failed 
Test control_ugwpv1 027 failed in run_test failed 
Test control_thompson 029 failed failed 
Test control_ras 028 failed failed 
Test control_thompson 029 failed in run_test failed 
Test control_ras 028 failed in run_test failed 
Test control_thompson_no_aero 030 failed failed 
Test control_thompson_no_aero 030 failed in run_test failed 
Test control_noahmp 031 failed failed 
Test control_noahmp 031 failed in run_test failed 
Test regional_control 032 failed failed 
Test regional_control 032 failed in run_test failed 
Test fv3_gsd 038 failed failed 
Test fv3_gsd 038 failed in run_test failed 
Test fv3_rrfs_v1alpha 039 failed failed 
Test fv3_rrfs_v1alpha 039 failed in run_test failed 
Test fv3_rap 040 failed failed 
Test fv3_rap 040 failed in run_test failed 
Test fv3_rrfs_v1beta 042 failed failed 
Test fv3_rrfs_v1beta 042 failed in run_test failed 
Test fv3_hrrr 041 failed failed 
Test fv3_hrrr 041 failed in run_test failed 
Test cpld_control_c384 004 failed failed 
Test cpld_control_c384 004 failed in run_test failed 
Test control_c192 014 failed failed 
Test control_c192 014 failed in run_test failed 
Test control_c384 015 failed failed 
Test control_c384 015 failed in run_test failed 
Test control_c384gdas 016 failed failed 
Test control_c384gdas 016 failed in run_test failed 
Please make changes and add the following label back:
cheyenne-intel-BL

**Comment by climbfuji** [comment]
Yes, definitely disk quota issue on Cheyenne:
```
Moving baseline 065 fv3_gsd_diag_debug files ....

 Moving sfcf000.tile1.nc .....cp: error writing '/glade/scratch/dtcufsrt/FV3_RT/REGRESSION_TEST_INTEL/fv3_gsd_diag_debug/sfcf000.tile1.nc': Disk quota exceeded
```
This also corrupted the baseline for GNU. Will cleanup and redo.

**Comment by BrianCurtis-NOAA** [comment]
> This also corrupted the baseline for GNU. Will cleanup and redo.

You may only need to clean up: `/glade/scratch/dtcufsrt/FV3_RT`

**Comment by MinsukJi-NOAA** [comment]
CI container has been updated to fms 2021.03. How can we bring these changes (two files under tests/ci) into this PR?
```
diff --git a/tests/ci/Dockerfile b/tests/ci/Dockerfile
index 8364c75..f95e5d1 100644
--- a/tests/ci/Dockerfile
+++ b/tests/ci/Dockerfile
@@ -1,4 +1,4 @@
-From noaaemc/ubuntu-hpc:v1.4b
+From noaaemc/ubuntu-hpc:v1.5b
 
 ENV HOME=/home/builder
 COPY --chown=builder:builder . $HOME/ufs-weather-model

diff --git a/tests/ci/ci.sh b/tests/ci/ci.sh
index 345a33b..57a96c1 100755
--- a/tests/ci/ci.sh
+++ b/tests/ci/ci.sh
@@ -81,7 +81,7 @@ elif [ $RUN == "true" ]; then
                 -e "RT_MACHINE=linux.gnu" -e "RT_COMPILER=gnu" \
                 -w "/home/builder/ufs-weather-model/tests" \
                 -v DataVolume:/home/builder/data/NEMSfv3gfs/input-data-20210528 \
-                --shm-size=512m --name my-container noaaemc/ubuntu-hpc:v1.4b \
+                --shm-size=512m --name my-container noaaemc/ubuntu-hpc:v1.5b \
                 /bin/bash -c "./utest -n ${TEST_NAME} -c ${TEST_CASE} -x"
 
   cd $GITHUB_WORKSPACE

```

**Comment by junwang-noaa** [comment]
@bensonr Would you please commit the code changes from Minsuk to your branch? Thanks

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/720092158/20210826110011/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_53807
Please make changes and add the following label back:
orion-intel-BL

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: gaea
Compiler: intel
Job: RT
Repo location: /lustre/f2/pdata/ncep/emc.nemspara/autort/pr/720092158/20210826164508/ufs-weather-model
Please manually delete: /lustre/f2/scratch/emc.nemspara/FV3_RT/rt_9846
Test control 017 failed failed 
Test control 017 failed in run_test failed 
Test control_merra2 034 failed failed 
Test control_merra2 034 failed in run_test failed 
Test regional_restart 044 failed failed 
Test regional_restart 044 failed in run_test failed 
Please make changes and add the following label back:
gaea-intel-RT

**Comment by MinsukJi-NOAA** [comment]
All tests that failed on gaea were timed out. For example, control_merra2 ran for 15 hours instead of 24 hours. I will manually re-run those 3 tests.

**Comment by junwang-noaa** [comment]
Thanks, Minsuk.

**Comment by junwang-noaa** [comment]
@BrianCurtis-NOAA The orion baseline was created successfully, but the baseline was not copied to the /work/noaa/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20210825/INTEL, would you please manually copy them, then change permission so that all the developers can see it? Thanks

**Comment by BrianCurtis-NOAA** [comment]
> @BrianCurtis-NOAA The orion baseline was created successfully, but the baseline was not copied to the /work/noaa/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20210825/INTEL, would you please manually copy them, then change permission so that all the developers can see it? Thanks

Sure thing. I've started the rsync. Once completed I'll add the orion intel RT label.

**Comment by junwang-noaa** [comment]
@bensonr the wcoss dell/cray log files are on hera at:
```
/scratch1/NCEPDEV/stmp2/Jun.Wang/wcosslog
RegressionTests_wcoss_cray.log  RegressionTests_wcoss_dell_p3.log
```
Please commit them to your branch

**Comment by bensonr** [comment]
@junwang-noaa - wcoss regression logs are in place

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: jet
Compiler: intel
Job: BL
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/720092158/20210826133015/ufs-weather-model
Please manually delete: /lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_17906
Baseline creation and move successful
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/720092158/20210826174959/ufs-weather-model
Please manually delete: /lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_51597
Test control_2threads 018 failed failed 
Test control_2threads 018 failed in run_test failed 
Please make changes and add the following label back:
jet-intel-BL

**Review by climbfuji — APPROVED** [review]
Approved contingent on jet.intel tests passing (and the usual submodule/.gitmodules procedure).

**Comment by BrianCurtis-NOAA** [comment]
With current Jet issues, you may want to skip it for this PR.

**Comment by junwang-noaa** [comment]
I have jet log file ready, but now jet is not accessible. Will attach to the PR when jet is back.

**Review by junwang-noaa — APPROVED** [review]
